src/05-derivatives/01-introduction.md

```diff
@@ -28,6 +28,76 @@ in [Derived dataset and pipeline description][derived-dataset-description].
     to ensure that the values are still relevant and appropriate to the type of
     output data.
 
+## Model-based grouping of derivatives
+
+Here, a *model* is defined as a process via which parameters that describe
+the data are fit/derived. This could include, but is not limited to things
+like:
+
+1.  The beta weights for different kinds of events in an event-related design
+    in fMRI.
+
+1.  The six components of the diffusion tensor in DTI.
+
+1.  Etc.
+
+Model-based derivatives SHOULD be saved in a directory named
+`model-<modelname>` that is placed under the datatype from which the model
+was derived. A metadata file, `models.tsv` is OPTIONAL, accompanied by
+a `models.json` file that is REQUIRED, if and only `models.tsv` is present.
+The `models.json` is a dictionary for the `models.tsv` columns.
+The purpose of the RECOMMENDED `models.tsv` file is to describe properties of the models
+such as the datatype from which they are derived and a human-readable description of the
+model. An additional metadata file, `model-<modelname>.json` is REQUIRED and describes the metadata for model fitting procedures, etc. for each model.
```

Comment:
I would leave the model metadata open for definition after the folder idea is accepted. I agree that we have at BEP016 a proposal that potentially generalizes across modalities, but would not slow down this new folder proposal:
```suggestion
model.
```